_includes/header.html

```diff
@@ -21,7 +21,7 @@
         <div class="trigger">
           {% for path in page_paths %}
             {% assign my_page = site.pages | where: "path", path | first %}
-            {% if my_page.title %}
+            {% if my_page.title and !my_page.exclude != true %}
```

Comment:
Where in the docs would I put that? I can't find a section that seems a good fit to me (and I'm a Jekyll newbie, so a bit lost anyway).